dfastmi/io/AReach.py

```diff
@@ -55,6 +55,17 @@ def __init__(self, reach_name: str = "Reach", reach_config_key_index: int = 1):
         self._name = reach_name
         self._config_key_index = reach_config_key_index
 
+    def __hash__(self):
+        return hash((self._name, self._config_key_index))
+
+    def __eq__(self, other):
+        if not isinstance(other, AReach):
+            return False
+        return (
+            self._name == other._name
```

Comment:
The == should only check on name and index, not check on other values of the Areach object?
e.g. normal_width, ucritical, qstagnant, parent_branch